API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
+description | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+url | UrlField  
+difficulty | CharField (choices)  
+duration | CharField (choices)  
+**Skills** | *:*  
+**WebLitSkills** | *:*  
+**Topic** | 1:*  
+**LessonPlan** | *:*  
+**locale** | CharField (choices) (pycountry)  
+translation_of | ‘self’  
+
+
+
+<pre>Skills</pre>
+name | CharField  
+description | TextField  
+**Competencies** | 1:*  
+image_url | UrlField  
+image_retina_url | UrlField  
+
+<pre>WebLitSkills</pre>
+name | CharField  
+short_name | CharField  
+description | TextField  
+**Competencies** | 1:*  
+**Skills** | *:*  
+**Topic** | 1:*  
+
+<pre>Topic</pre>
+name | CharField  
+description | TextField  
+
+<pre>Competencies</pre>
+text | TextField  
+
+<pre>LessonPlan</pre>
+title | CharField  
```

Comment:
Now that I realize I had Activity and LessonPlan backwards in my head, this makes more sense. Comment above still stands, though, just with the terms reversed. If we could provide a generated sum total for the collection of Activities that make up the LessonPlan, that would be useful.